newcomers-tutorial.md

```diff
@@ -246,42 +240,39 @@ can specify multiple tests, separated by whitespace:
 
 ## Finding a task to work on
 
-Since you came here, you probably have an idea what to work on already - but if
-not, do check the [issues tagged
+You probably have an idea what you want to work on already - but if not,
+do check the [issues tagged
 "newcomer"](https://gitlab.haskell.org/ghc/ghc/issues?scope=all&utf8=%E2%9C%93&state=opened&label_name[]=newcomer) -
 these are issues we expect to be "low hanging fruit".
-Of course, we can't ever be sure of how hard a task is before doing it, so
-apologies if one of these is too hard.
+
+Of course, we can't be sure how hard a task is before doing it, so
+apologies in advance if one of these is too hard.
 
 Either way, **if you are going to work on something, make sure a ticket exists
 for it**. This is essential for all sorts of things, but most importantly,
 attaching things to tickets makes sure they don't get lost. So if there is no
-ticket for it yet, do file one. The issue tracker is [right
-here](https://gitlab.haskell.org/ghc/ghc/issues/new).
+ticket for it yet, please use the [issue tracker](https://gitlab.haskell.org/ghc/ghc/issues/new) to do so.
 
-Apart from that, you are encouraged to ask for a starting point on IRC or the
-`ghc-devs` [mailing list](mailing-lists-and-irc). There someone familiar with
-the process can help you find a ticket that matches your expertise and help you
-when you get stuck.
+We'd like to encouraged you to ask for a starting point on IRC or the
+`ghc-devs` [mailing list](mailing-lists-and-irc). You'll find someone familiar
+with the process who can help you find a ticket that matches your expertise and you'll also find people who can help you when you get stuck.
 
 ## Working with the code
 
-GHC is an old codebase, with plenty of historic quirks to it, and the things it
-does are intrinsically complex, so it can be quite overwhelming. Don't let that
-discourage you, and don't hesitate to ask questions.
+GHC is a decades-old codebase, with plenty of historic quirks to it, and the things it does are intrinsically complex, so it can be quite overwhelming.
+Please don't let that discourage you, and don't hesitate to ask questions.
 
 ### Coding Style
 
 #### General
 
 GHC doesn't follow a uniform coding style across all modules, and we do not
-enforce any particular one. The general rule is to stick to the same coding
+enforce any in particular. The general rule is to stick to the same coding
 style as is already used in the file you're editing. If you must make stylistic
-changes, commit them separately from functional changes, so that someone
-looking back through the change logs can easily distinguish them.
+changes, commit them separately from functional changes, so someone looking
+back through the change logs can easily distinguish them.
 
-It's much better to write code that is transparent than to write code that is
-short.
+In general, it's much better to write code that is transparent than to write code that is short.
```

Comment:
- "In general" is practically empty fluff, doesn't add much value. I'd keep the original.
- Diff noise, suggest putting line break back.